roles/contiv/tasks/delete_contiv.yml

```diff
@@ -0,0 +1,31 @@
+- name: delete contiv
```

Comment:
moved (and replied to this) earlier ... just adding reply here now also directly under the original comment for completeness.